app/javascript/admin/dashboard/index.js

```diff
@@ -0,0 +1,64 @@
+import Chart from 'chart.js';
+
+function createChart(id, title, dataSource) {
+  var wrapper = document.getElementById('wrapper');
+  var canvas = document.createElement('canvas');
```

Comment:
have you verified that this use of canvas is not blocked by Brave when fingerprinting protection is set to block all?

Reply:
This is internal tool, and yes it's not blocked.